Doc/Makefile

```diff
@@ -38,6 +38,8 @@ help:
 	@echo "  serve      to serve the documentation on the localhost (8000)"
 
 build:
+	-mkdir build
+	blurb merge -f build/NEWS
```

Comment:
To get around the PATH problem, suggest:
1. Adding
`BLURB        = $(PYTHON) -m blurb`
after line 8 (`SPHINXBUILD  = sphinx-build`) in the definitions at the top of the file and then
2. changing the line above to:
`$(BLURB) merge -f build/NEWS`
That way, if users do a `make venv ; make html PYTHON=venv/bin/python`, `blurb` will be found there but, if they need to use a different version of `blurb`, they can override it: `make html BLURB=/usr/local/bin/blurb`.